src/DataSkimmer.jl

```diff
@@ -57,14 +57,17 @@ struct CategoricalColumn
     type::Type
     n_missing::Int64
     completion_rate::Float64
+    n_unique::Int64
 
     function CategoricalColumn(data, column_name)
         n_missing = count(ismissing, Tables.getcolumn(data, column_name))
+        n_unique = length(unique(skipmissing(Tables.getcolumn(data, column_name))))
```

Comment:
I think skipping the missing values might confuse the user. I would expect a column with `"true"`, `"false"` and `missing` values to have three different values. Let me know if you think something different though.

Reply:
Here's the `skim` output for a categorical column from a current analysis.

![categorical_column_skim](https://user-images.githubusercontent.com/16549651/125986226-4e1dd353-8483-4d5e-9109-5d1a7d9af5ae.png)

I know the column will have a 143 'levels' of this category. If the missing column is non-zero, I know there would also be a 'missing' level.

In my workflow, I'm often reporting out how many of several Departments, Locations, etc. are present in a dataset. For these purposes, one wouldn't add 'missing' as a level. Keeping these concepts separate helps that type of workflow. Otherwise, for each column, one would need to read, for example, 143 levels, check to see if missings were present, and if so, subtract one from the count for the report.

Let me know what you think.

Reply:
Thanks for explaining, I wasn't sure about this, but I'm convinced now. I'll merge it now.